### PR TITLE
Misc UI improvements

### PR DESCRIPTION
**Pull request by Sophist-UK**
This is an interim PR with some minor changes.

I am submitting this now as the next step are major changes to elements.py to add context menu help text and tweak right click menu functionality, and to avoid a headache rolling these changes forwards into 0.8.x these would probably be better once the restructuring done in 0.8 has been retrofitted to 0.7 and rolled forward.

### CODE REVIEW

File: meerk40t/core/elements.py [hunk 1]
```diff
@@ -1892,6 +1900,9 @@ def inner(node, **ik):
             inner.values = [0]
             registered_name = inner.__name__
 
+            if help: # DEBUGGING
+                print(name,registered_name,help)
```
Comment (tatarize) [inline meerk40t/core/elements.py:1904]: Debug code.

File: meerk40t/gui/wxmmain.py [hunk 2]
```diff
@@ -163,7 +164,17 @@ def __init__(self, *args, **kwds):
         self.main_menubar = wx.MenuBar()
         self.__set_menubars()
 
-        self.main_statusbar = self.CreateStatusBar(3)
+        self.main_statusbar = self.CreateStatusBar(4)
+        self.main_statusbar.SetStatusStyles([wx.SB_SUNKEN] * self.main_statusbar.GetFieldsCount())
+        self.main_statusbar.SetStatusWidths([-1] * self.main_statusbar.GetFieldsCount())
+        self.SetStatusBarPane(0)
+        self.main_statusbar.SetStatusText(_("Status..."), 0)
+        self.Bind(wx.EVT_MENU_OPEN, self.on_menu_open) # ToDo DELETE - only for testing
```
Comment (tatarize) [inline meerk40t/gui/wxmmain.py:172]: Testing

File: meerk40t/gui/wxutils.py [hunk 3]
```diff
@@ -113,7 +115,9 @@ def specific(event=None):
                 menu.AppendSeparator()
             if submenu_name is not None:
                 submenu = wx.Menu()
-                menu.AppendSubMenu(submenu, submenu_name)
+                if func.help:
+                    print("menu", submenu_name, func.help)
```
Comment (tatarize) [inline meerk40t/gui/wxutils.py:119]: debug

**Review by tatarize — CHANGES_REQUESTED** [review]

File: meerk40t/core/elements.py (hunk 1 above)
Comment (Sophist-UK) [reply meerk40t/core/elements.py]: Removed.

File: meerk40t/gui/wxutils.py (hunk 3 above)
Comment (Sophist-UK) [reply meerk40t/gui/wxutils.py]: Done.

File: meerk40t/gui/wxmmain.py (hunk 2 above)
Comment (Sophist-UK) [reply meerk40t/gui/wxmmain.py:172]: Actually - this needs to stay - it's just the comment that needs to go.

**Review by tatarize — APPROVED** [review]

**Comment by Sophist-UK** [comment]
Ok - I'll wait for the elements.py restructuring in main to be done. I'll review and approve the PR as quickly as I can once it appears.

**Pull request merged**